File: Período_1/Lista 7/L7_Q4.c
/*******************************************************************
*   Faça um programa que faça 5 perguntas para uma pessoa sobre 
*   um crime. O programa deve no final emitir uma classificação
*   sobre a participação da pessoa no crime. Se a pessoa responder
*   positivamente a 2 questões ela deve ser classificada como 
*   “Suspeita”, entre 3 e 4 como “Cúmplice” e 5 como “Assassino“.
*   Caso contrário, ele será classificado como “Inocente “.
*******************************************************************/
#include <stdio.h>

int main(void){

    char p1;
    int n = 0;
    
    printf("Responda as seguintes perguntas com \"s\" para SIM e \"n\" para NAO\n");
    printf("Telefonou para a vitima? ");
    scanf(" %c", &p1);
    p1 == 's'? n++ : n == n;
    printf("Esteve no local do crime? ");
    scanf(" %c", &p1);
    p1 == 's' ? n++ : n == n;
    printf("Mora perto da vitima? ");
    scanf(" %c", &p1);
    p1 == 's' ? n++ : n == n;
    printf("Devia para a vitima? ");
    scanf(" %c", &p1);
    p1 == 's' ? n++ : n == n;
    printf("Ja trabalhou com a vitima? ");
    scanf(" %c", &p1);
    p1 == 's' ? n++ : n == n;

    if(n < 2)
        printf("Inocente.\n");
    else if(n < 3)
        printf("Suspeita.");
    else if (n < 5)
        printf("Cumplice.");
    else
        printf("Assasino.");
    
}

File: Período_1/Lista 7/Python/L7_Q4.py
cont = 0
respostas = []
print("\nResponda s ou n para as perguntar:")
respostas.append(input("Telefonou para a vítima? "))
respostas.append(input("Esteve no local do crime? "))
respostas.append(input("Mora perto da vítima? "))
respostas.append(input("Devia para a vítima? "))
respostas.append(input("Já trabalhou com a vítima? "))

cont = respostas.count('y')
print()
if cont == 5:
    print("Assassino")
elif cont >= 3:
    print("Cúmplice")
elif cont == 2:
    print("Suspeita")
else:
    print("Inocente")

print()